core/src/main/java/org/togglz/core/Feature.java

```diff
@@ -1,37 +1,31 @@
 package org.togglz.core;
 
+import org.togglz.core.context.FeatureContext;
+
 /**
- * 
  * <p>
  * This interface represents a feature and is typically implemented by the feature enum.
  * </p>
- * 
- * <p>
- * Usually it makes sense to implement the following method which allows a very easy way to check the status of a feature.
- * </p>
- * 
- * <pre>
- * public boolean isActive() {
- *     return FeatureContext.getFeatureManager().isActive(this);
- * }
- * </pre>
- * 
- * <p>
- * Please note that in Togglz 2.0 the <code>isActive()</code> method isn't defined in the interface any more. So when updating
- * to Togglz 2.0, you will have to remove the {@link Override} annotation from your implementation of the method.
- * </p>
- * 
+ *
  * @author Christian Kaltepoth
- * 
+ * @author Michał Rowicki
```

Comment:
It's no need to add your name. Of course you can do that if you want, but I am not requiring this :) so if you want to get contacted, you can add your name here of course :)

Reply:
I'm just get used to it from spring-framework repos, anyway I'll remove it :)

Reply:
you will be in the hall of fame of committers anyway :)

Reply:
I'm aware of `git blame` 🤣

Reply:
haha :-D